kv/etcd/store.go

```diff
@@ -522,8 +522,13 @@ func newValue(val []byte, ver, rev int64) *value {
 	}
 }
 
-func (c *value) isNewer(other *value) bool {
-	return c.Rev > other.Rev
+func (c *value) IsNewer(other kv.Value) bool {
+	othervalue, ok := other.(*value)
```

Comment:
So does that mean `kv.Value` can be some type other than a `*value`? We should only keep the `ok` check if and only if the `kv.Value` can be more than one type.